docker-compose.yml

```diff
@@ -9,7 +9,36 @@ services:
       dockerfile: ./docker/Dockerfile
     ports:
       - "${PORT:-5000}:${PORT:-5000}"
+    volumes:
+      - .:/server
     network_mode: "bridge"
     environment: # forward these from shell to the container at runtime
       - DEBUG
       - PORT
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgres
+      - POSTGRES_DB=postgres
+    links:
+      - postgres
+    depends_on:
+      - postgres
+  
+  postgres:
+    image: postgres:12.0
+    container_name: postgres
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./postgres_data:/var/lib/postgresql/data
+      - ./summarizer_server/sql/init.sql:/docker-entrypoint-initdb.d/01-init.sql
```

Comment:
Oh yea, sure that could work :).

> I'm kind of confused what alembic does here, if the postgres container already runs the sql scripts automatically

Oh that's a good point actually, I should be programmatically creating the tables using SQA instead of using sql directly (my bad), so we'd either:
1. Use sql scripts and not alembic, then migrations are manual (alter statements, etc.)
2. Use alembic and not sql scripts (get help with migrations, less chance for errors in SQL)

I'm leaning towards option 2 actually, wdyt?

Reply:
Yea option 2 sounds pretty good. That would mean we have a pretty big dependency on sqlalchemy, but that's probably fine.